src/main/java/ezvcard/io/json/JCardRawReader.java

```diff
@@ -86,11 +99,13 @@ public void readNext(JCardDataStreamListener listener) throws IOException {
 		this.listener = listener;
 
 		//find the next vCard object
-		JsonToken prev = null;
+		JsonToken prev = parser.getCurrentToken();
 		JsonToken cur;
 		while ((cur = parser.nextToken()) != null) {
 			if (prev == JsonToken.START_ARRAY && cur == JsonToken.VALUE_STRING && "vcard".equals(parser.getValueAsString())) {
 				break;
+			} else if (strict) {
+				throw new JCardParseException(JsonToken.VALUE_STRING, cur);
```

Comment:
That would be an invalid jcard, right? What kind of exception would you throw for that? Maybe add a String constructor to JCardParseException?

Another thing I've realized is that there might actually be a JsonToken.VALUE_NULL if the vcard field is null. I would expect that checking for that immediately after reading the first token and returning immediately (before the vcard is initialized in the listener with listener.beginVCard()) should be safe enough, like:
```java
JsonToken prev = parser.getCurrentToken();
if (prev == JsonToken.VALUE_NULL) {
	return;
}
```
Because if a string begins with `null`, there's nothing that could come after it and still be valid json.